src/main/java/com/example/home_work_6/Car.java

```diff
@@ -0,0 +1,24 @@
+package com.example.home_work_6;
+
+public class Car {
+
+    public void start(){
+        startElectricity();
+        startCommand();
+        startFuelSystem();
+
```

Comment:
Would be nice to have some information if the engine has started, right? 
Could you please add println() statement with the comment "Engine is started." ?